pom.xml

```diff
@@ -47,12 +46,95 @@
             <artifactId>hadoop-client</artifactId>
             <version>2.2.0</version>
         </dependency>
+
+	<dependency>
+    	    <groupId>org.apache.hadoop</groupId>
+    	    <artifactId>hadoop-hdfs</artifactId>
+    	    <version>2.2.0</version>
+	</dependency>
+
+        <dependency>
+    	    <groupId>org.apache.hadoop</groupId>
+    	    <artifactId>hadoop-hdfs</artifactId>
+    	    <version>2.2.0</version>
+    	    <classifier>tests</classifier>
+	</dependency>
+
+	<dependency>
+    	    <groupId>org.apache.hadoop</groupId>
+    	    <artifactId>hadoop-mapred</artifactId>
+    	    <version>0.22.0</version>
+	</dependency>
+
+	<dependency>
+    	    <groupId>org.apache.hadoop</groupId>
+    	    <artifactId>hadoop-core</artifactId>
+    	    <version>1.1.2</version>
+	</dependency>
+
+	<dependency>
+    	    <groupId>org.apache.hadoop</groupId>
+    	    <artifactId>hadoop-minicluster</artifactId>
+    	    <version>2.2.0</version>
+	</dependency>
+
+	<dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client</artifactId>
+            <version>2.2.0</version>
+        </dependency>
+
+	<dependency>
+    	    <groupId>commons-configuration</groupId>
+    	    <artifactId>commons-configuration</artifactId>
+    	    <version>1.10</version>
+	</dependency>
+
+	<dependency>
+    	    <groupId>xerces</groupId>
+    	    <artifactId>xercesImpl</artifactId>
+    	    <version>2.9.1</version>
+	</dependency>
+		
+	<dependency>
+    	    <groupId>xalan</groupId>
+    	    <artifactId>xalan</artifactId>
+    	    <version>2.7.1</version>
+	</dependency>
+
+	<dependency>
+    	    <groupId>org.apache.hive</groupId>
+    	    <artifactId>hive-common</artifactId>
+    	    <version>1.2.1</version>
+	</dependency>
+
+	<dependency>
+    	   <groupId>org.apache.hive</groupId>
+    	   <artifactId>hive-service</artifactId>
+    	   <version>1.2.1</version>
```

Comment:
If we use Hive 1.2.1, then we should state it in the README, because the current README says "Beetest has been tested on Hadoop 2.2.0 (YARN) and Hive 0.12.0.".